projects/packages/forms/src/blocks/field-date/edit.js

```diff
@@ -39,7 +39,6 @@ export default function DateFieldEdit( props ) {
 		template,
 		templateLock: 'all',
 	} );
-
```

Comment:
We can roll back this change.

Reply:
I'm not so sure. I think everything depends on this change. I don't want to see the whole house of cards come crashing down.

Reply:
You're right.

This is hilarious and tragic at the same time. 🤣 

Hilaragic